common/crypto/Makefile

```diff
@@ -7,15 +7,22 @@ include $(TOP)/build.mk
 
 BUILD_DIR := _build
 
+PDO_ENCLAVE_CODE_SIGN_PEM := "pdo_enclave_code_sign.pem"
+
 $(BUILD_DIR):
 	@if [ ! -d $(BUILD_DIR) ]; then \
 		mkdir -p $(BUILD_DIR) && \
-		cd $(BUILD_DIR) && \
-		cmake ./..; \
-	fi
+        cd $(BUILD_DIR) && \
+		openssl genrsa -3 -out $(PDO_ENCLAVE_CODE_SIGN_PEM) 3072 && \
+        export PDO_ENCLAVE_CODE_SIGN_PEM=`readlink -f $(PDO_ENCLAVE_CODE_SIGN_PEM)` && \
```

Comment:
We already have an enclave signer for FPC, can't we just use that and point PDO_ENCLAVCE_CODE_SIGN_PEM and ultimately we might want only a single signer key which could be configured? (admittedly, not really important for test programs but still a bit confusing)

PS: Looking at FPC code i realize that we do that kindof ugly in ../../ecc_enclave/enclave/CMakeLists-common-app-enclave.txt and ../../tlcc_enclave/enclave/CMakeLists.txt where we unconditionally create separate new keys.  So this seems more like a separate issue to be opened ...

Reply:
PDO_ENCLAVCE_CODE_SIGN_PEM is required in PDO.
Anyway:
1) we don't need configurability for the signer key
2) we should not have dangling keys in the repo.
It is for (1) (2) that "I" removed those keys in FPC and coded their creation just and only when we need them at runtime. 
For the same reasons, I was actually thinking of removing the key in PDO too -- additionally getting rid of unnecessary complexity.

Reply:
I've created now issue #368 for that ...

Reply:
I realize that we need a defined PDO_ENCLAVE_CODE_SIGN_PEM variable pointing to a key.  If we want a secure version of FPC, we _do need_ though a configurable version of that key (see #368).  But certainly hasn't to be done now and even less so in this PR.  (I just first we already had a centralized version in which case we should have reused the same one instead of creating a new one ...)